cognoml/tests/unit_tests/test_utils.py

```diff
@@ -0,0 +1,11 @@
+import os
+import mock
+from cognoml import utils 
+
+@mock.patch('os.mkdir')
+@mock.patch('os.path.exists')
```

Comment:
These mocks look cool. So what exactly does it do to the `os.mkdir`/ `os.path.exists` functions? Trying to understand.